src/promptflow-evals/promptflow/evals/evaluators/_content_safety/common/evaluate_with_rai_service.py

```diff
@@ -9,10 +9,12 @@
 import requests
 from azure.core.credentials import TokenCredential
 from azure.identity import DefaultAzureCredential
-from constants import EvaluationMetrics, RAIService, Tasks
-from utils import get_harm_severity_level
-
-from promptflow.core import tool
+try:
```

Comment:
What is ther reason for having this ?

Reply:
Workaround until I determine how to fix the import issue that flex flows have when loaded.

If used directly, this flex flow runs as expected, but it saved and loaded (as happens in the unit test "test_load_and_run_evaluators"), relative import statements for files loaded by the PF client fail. This makes it so that at least our builtin evaluators still work.